recipes/configs/dev/3B_full_grpo.yaml

```diff
@@ -3,7 +3,7 @@
 #
 # This config assumes that you've run the following command before launching
 # this run:
-#   tune download meta-llama/Llama-3.2-3B --output-dir /tmp/Llama-3.2-3B --ignore-patterns "original/consolidated.00.pth"
+#   tune download Qwen/Qwen2.5-3B --output-dir /tmp/Qwen2.5-3B
```

Comment:
This shouldn't be changed, right? Isn't this still the sync version using Llama 3.2 3B?